### DISCUSSION TITLE
求助。我无法在matebook ego上使用arm64的alist.

### DISCUSSION THREAD

**Discussion by dddclearlove** [AlistGo/alist#5799] [Q&A]
![image](https://github.com/alist-org/alist/assets/67955849/8cd50b71-ea60-438f-9964-e5699912da54)

处理器是Snapdragon (TM) 8cx Gen 3，x64时转移运行成功，但无法运行arm64.会出现图片情况

**Comment by im-dashan** [comment]
我可以帮助你，我遇到过这种情况
1.你可能更改了配置里的数据库配置，请先运行数据库后再启动alist
2.你没有开放对应端口，

我上次遇到这个问题是因为更换了数据库，但是数据库没有打开，运行alist机会卡到这一步

**Reply by dddclearlove** [reply]
谢谢，具体该怎么做呢？我是在一台全新电脑上使用的，不知道该如何启动数据库与开放对应端口

**Comment by im-dashan** [comment]
你先告诉我，你有没有更换数据库，用的什么终端，架构，个人电脑还是服务器，亦或者路由器

dddclearlove ***@***.***> 于2024年1月5日周五 14:58写道：

**Reply by dddclearlove** [reply]
我使用的是个人电脑,arm处理器，win on arm系统。数据库这方面我不太了解，就是下载好后执行便失败了

**Comment by im-dashan** [comment]
你可以在网址，yyds.cam,,获得我的联系方式，即时我会帮助你

大山 ***@***.***> 于2024年1月5日周五 22:55写道：

**Comment by im-dashan** [comment]
你可以试试解除./alist server 的占用端口，或者试试在 conf.json 的配置文件编辑数据库的端口或者试试先启动数据库 mysql
或者 db ，如果你有更换数据库的话，建议你重新在 alist 重新下载alist-Windows-arm64.zip 或者 exe，

dddclearlove ***@***.***>于2024年1月5日 周五23:55写道：
